src/site/_includes/macros/post-card.njk

```diff
@@ -14,38 +14,23 @@
 {% set isDraft = 'w-card--draft' if data.draft %}
```

Comment:
This class does not exist in the new design system and will not load in context of this partial.

I think it was intended to hide the card is he post is a draft (?). Please check and if this is the case, use hidden-yes or visibility-hidden utility classes.